install_config/upgrading/manual_upgrades.adoc

```diff
@@ -899,6 +899,114 @@ For Pacemaker HA multiple master deployments:
 After the service restarts, the certificate update is complete.
 // end::30to31updatingcerts[]
 
+[[manual-upgrading-logging-components]]
+== Upgrading Logging Components
+
+If you have previously link:../../install_config/aggregate_logging.html[deployed
+logging components] and want to upgrade your EFK stack with new images, you must
+take the following steps to safely upgrade with minimal disruption to your log
+data.
+
+[NOTE]
+====
+The following steps apply when upgrading from {product-title}
+ifdef::openshift-origin[]
+1.1 to 1.2.
+endif::[]
+ifdef::openshift-enterprise[]
+3.1 to 3.2.
+endif::[]
+====
+
+. Scale down your Fluentd instances to 0:
++
+----
+$ oc scale dc/logging-fluentd --replicas=0
+----
++
+Wait until they have properly terminated. This helps prevent loss of data by
+giving them time to properly flush their current buffer and send any logs they
+were processing to Elasticsearch.
+
+. Scale down your Kibana instances:
++
+----
+$ oc scale dc/logging-kibana --replicas=0
+$ oc scale dc/logging-kibana-ops --replicas=0 <1>
+----
+<1> If applicable.
+
+. Once your Fluentd and Kibana pods are confirmed to be terminated, you can
+safely scale down the Elasticsearch pods:
++
+----
+$ oc scale dc/logging-es-<unique_name> --replicas=0
+$ oc scale dc/logging-es-ops-<unique_name> --replicas=0 <1>
+----
+<1> If applicable.
+
+. After your Elasticsearch pods are confirmed to be terminated, pull in the
+latest EFK images using the same procedure described in
+link:../../install_config/upgrading/manual_upgrades.html#importing-the-latest-images[Importing
+the Latest Images], replacing the *openshift* project with the project where
+logging was installed. For example:
++
+----
+$ oc import-image -n logging <imagestream>
+----
+
+. With the latest images in your repository we can now rerun the deployer to generate
+any missing or changed features.
+
+.. First, ensure that your OAuth client has been deleted:
++
+----
+$ oc delete oauthclient --selector logging-infra=support
+----
+
+.. Then, proceed to follow the same steps as done previously in
+link:../../install_config/aggregate_logging.html#deploying-the-efk-stack[Deploying
+the EFK Stack]. After the deployer completes, re-attach your persistent volumes
+you were using previously.
+
+. Next, scale Elasticsearch back up incrementally so that the cluster has time
+to rebuild:
```

Comment:
Is 'incrementally' the right word here? I'm reading this as "keep incrementally adding more replicas" but I'm not sure if that's the intention. Maybe the one command in the example is putting me off.